### PR TITLE
macOS: reduce latency

### PR DESCRIPTION
**Pull request by letmaik**
Resolves a left-over task from #16.

@JayFoxRox Could you do me one last favour for the mac variant? I changed the run loop message processing to process exactly one message each time without blocking. I think this is fine, but it would be good to check it didn't break anything. 

You can download the wheel from the CI artifacts: https://github.com/letmaik/pyvirtualcam/actions/runs/627620668

### CODE REVIEW

File: pyvirtualcam/native_macos/main.mm [hunk 1]
```diff
@@ -37,10 +37,10 @@
 static uint32_t cam_fps_den;
 
 
-static void YfromRGB(uint8_t* y, uint8_t r, uint8_t g, uint8_t b) {
+static void YfromRGB(uint8_t* y, double r, double g, double b) {
     *y = (uint8_t)( 0.257 * r + 0.504 * g + 0.098 * b +  16);
 }
-static void UVfromRGB(uint8_t* u, uint8_t* v, uint8_t r, uint8_t g, uint8_t b) {
+static void UVfromRGB(uint8_t* u, uint8_t* v, double r, double g, double b) {
```
Comment (JayFoxRox) [inline pyvirtualcam/native_macos/main.mm:43]: This used to be a `double` originally, but I was concerned with performance.
I can think of a couple of common platforms where `double` is much slower than `float` (or fixed point).

Comment (letmaik) [reply pyvirtualcam/native_macos/main.mm:43]: I see your point, but Macs in general are quite powerful so it's probably not an issue, don't you think? I'm happy to revert it if there is no visible difference in quality.

Comment (letmaik) [reply pyvirtualcam/native_macos/main.mm]: After some experiments on Windows with animated gradients I couldn't notice a quality difference, so I reverted this change again.

**Comment by letmaik** [comment]
@JayFoxRox I merged the setup.py part of this PR in another PR, so it now only contains the change to the loop message processing.

**Comment by letmaik** [comment]
I've verified the fix as I had access to a mac. It does reduce the latency quite a bit, the webcam filter sample benefits a lot from it. There seems to be minor stuttering (frame drops?) every now and then which seems better now as well, but I have a feeling for it to be fully gone the implementation has to be changed somehow, maybe using threading. I only noticed this with the video sample when I played a smoothly rotating globe, otherwise you wouldn't notice it.

**Pull request merged**